css/layout.css

```diff
@@ -0,0 +1,65 @@
+/* layout.css */
+body {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: #f0f0f0;
+}
+
+header {
+  background-color: #d32f2f;
+  color: white;
+  padding: 1rem;
+  text-align: center;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 1rem;
+}
+
+nav button {
+  background-color: #f44336;
+  border: none;
+  color: white;
+  padding: 0.5rem 1rem;
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
+
+nav button:hover {
+  background-color: #e53935;
+}
+
+main {
+  flex: 1;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
There is not really much point of having your `main` element as a `flex column` You are using `section` elements which are `block` level by default and start on new lines effectively stacking in a column.